src/ncbitaxon.py

```diff
@@ -24,24 +25,27 @@
 related_synonym = "oboInOwl:hasRelatedSynonym"
 broad_synonym = "oboInOwl:hasBroadSynonym"
 
+# See OMO properties at
+# https://github.com/information-artifact-ontology/ontology-metadata/blob/master/src/templates/annotation_properties.tsv
 predicates = {
-    "acronym": broad_synonym,
-    "anamorph": related_synonym,
-    "blast name": related_synonym,
-    "common name": exact_synonym,
-    "equivalent name": exact_synonym,
-    "genbank acronym": broad_synonym,
-    "genbank anamorph": related_synonym,
-    "genbank common name": exact_synonym,
-    "genbank synonym": related_synonym,
-    "in-part": related_synonym,
-    "misnomer": related_synonym,
-    "misspelling": related_synonym,
-    "synonym": related_synonym,
-    "scientific name": exact_synonym,
-    "teleomorph": related_synonym,
+    "acronym": (broad_synonym, "OMO:0003000", "abbreviation"),
```

Comment:
Maybe hold of on this until we have deliberated if we should have a separate "acronym" synonym type: https://github.com/information-artifact-ontology/ontology-metadata/issues/135

Reply:
The referenced discussio appears to be about whether a whole new annotation property should be made to indicate acronyms instead of using the existing infrastructure for annotating synonyms. For sure, an acronym is a type of abbreviation, and these have been used interchangeably across other ontologies.

Maybe in the mean time we should make an additional synonym type for acronym?

Also might mean we should try and get engagement on the discussion of nested synonym types since ideally we would have `OMO:acronym subPropertyOf OMO:abbreviation`. c.f. https://github.com/information-artifact-ontology/ontology-metadata/issues/151

Reply:
If this is a deal breaker for this PR, I can revert this line to using an *ad hoc* relation

Reply:
made a quick vote on slack to get a sense what others think. If the vote tends towards not separating them, by 3-1, let just merge this.

Reply:
Looks like overwhelming support for "they're close enough" based on https://obo-communitygroup.slack.com/archives/C01R2D66249/p1697739460459709

Reply:
Good outcome, good show.